web/src/components/ActivityCalendar.tsx

```diff
@@ -84,9 +84,10 @@ const ActivityCalendar = (props: Props) => {
                 "w-6 h-6 text-xs rounded-xl flex justify-center items-center border cursor-default",
                 "text-gray-400",
                 item.isCurrentMonth ? getCellAdditionalStyles(count, maxCount) : "opacity-60",
-                isToday && "border-zinc-400",
-                isSelected && "font-bold border-zinc-400",
-                !isToday && !isSelected && "border-transparent",
+                item.isCurrentMonth && isToday && "border-zinc-400",
+                item.isCurrentMonth && isSelected && "bg-success font-bold",
```

Comment:
Can we remove the `bg-success` from here. This is causing the date to indicate a memo is present event when there isn't one. With no memo there should only be a border set

expected:
![image](https://github.com/user-attachments/assets/b56b0e30-f8cc-4861-ac41-13eac0d97156)

this PR:
![image](https://github.com/user-attachments/assets/752524b0-53dd-4a07-8192-730a94c50159)

Reply:
You are correct. In my custom implementation, the logic for `isSelected` was modified, and the styles were also adjusted accordingly. Apologies for overlooking this and causing issues. Please review this new submission. #4153 

By the way, would it be possible to modify the way `selectedDate` is obtained? Based on the name of this variable, my understanding is that it should be set by selecting a date via a mouse click. However, obtaining it through `props` instead of state seems to prevent this variable from being updated by mouse clicks.